### PR TITLE
Made quantity and rate field as editable in new and edit invoice

### PR DESCRIPTION
**Pull request by gowsik-ragunath**
## Summary

Quantity and Rate fields in the invoice line item will be editable.

## Preview

https://www.loom.com/share/73dffec9b4884454bf95b63eadb860c7

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [X] I have manually tested all workflows
- [X] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by apoorv1316** [comment]
@gowsik-ragunath Why the qty. has so many decimal points. Didn't we fix it?
Screenshot 2022-08-04 at 2 00 20 PM

**Comment by gowsik-ragunath** [comment]
@apoorv1316 I think as we are performing division in quantity field ` const actualQuantity = (item.qty / 60) || (item.quantity / 60);` it is returning the values as it is with more 2 decimal values. I will update the quantity value to limit those values to 2 decimal places.

**Comment by gowsik-ragunath** [comment]
@apoorv1316 I have the decimal issue.

Screenshot 2022-08-04 at 2 13 56 PM

File: app/javascript/src/components/Invoices/common/NewLineItemRow/EditLineItems.tsx [hunk 1]
```diff
@@ -12,13 +12,14 @@ const EditLineItems = ({
 }) => {
 
   const strName = item.name || `${item.first_name} ${item.last_name}`;
-  const quantity = (item.qty / 60) || (item.quantity / 60);
+  const actualQuantity = ((item.qty / 60) || (item.quantity / 60)).toFixed(2);
```
[the remark this answers is not in the document]
Comment (gowsik-ragunath) [reply app/javascript/src/components/Invoices/common/NewLineItemRow/EditLineItems.tsx:15]: We are using this component in both the edit and generate invoice components.

I think in the edit invoice we are storing line item quantity in `qty`, and in generate invoice we are storing quantity in the `quantity` key.

**Review by apoorv1316 — APPROVED** [review]

**Pull request merged**